jwst/photom/photom.py

```diff
@@ -478,31 +490,35 @@ def calc_miri(self, ftab):
                 photom_corr = miri_imager.time_corr_photom(ftab.timecoeff[row], mid_time)
 
                 data = np.array(
-                    [(self.filter,
-                      self.subarray,
-                      ftab.phot_table[row]['photmjsr'] + photom_corr,
-                      ftab.phot_table[row]['uncertainty'])],
+                    [
+                        (
+                            self.filter,
+                            self.subarray,
+                            ftab.phot_table[row]["photmjsr"] + photom_corr,
+                            ftab.phot_table[row]["uncertainty"],
+                        )
+                    ],
                     dtype=[
                         ("filter", "O"),
                         ("subarray", "O"),
                         ("photmjsr", "<f4"),
-                        ("uncertainty", "<f4")
+                        ("uncertainty", "<f4"),
                     ],
                 )
                 fftab = datamodels.MirImgPhotomModel(phot_table=data)
                 self.photom_io(fftab.phot_table[0])
             except AttributeError:
                 # No time-dependent correction is applied
-                log.info(" Skipping MIRI imager time correction. Extension not found in the reference file.")
+                firstpart = " Skipping MIRI imager time correction."
+                secondpart = " Extension not found in the reference file."
+                log.info(firstpart + secondpart)
                 self.photom_io(ftab.phot_table[row])
```

Comment:
You should be able to just do something like
```suggestion
                log.info(
                    " Skipping MIRI imager time correction."
                    " Extension not found in the reference file."
                )
                self.photom_io(ftab.phot_table[row])
```

Reply:
Yes, This works, thank you!  I had a comma between the strings, and that caused a unit test to fail